lib/_http_server.js

```diff
@@ -213,8 +213,25 @@ ServerResponse.prototype.writeHeader = function() {
 };
 
 
-function Server(requestListener) {
-  if (!(this instanceof Server)) return new Server(requestListener);
+var TLS_ARGS = ['pfx', 'key', 'passphrase', 'cert', 'ca', 'crl', 'ciphers',
```

Comment:
The minimum would be either `pfx` or both `key` and `cert`. Certainly better than the full list in terms of maintainability.